src/frontend/src/icp/components/transactions/IcTransactions.svelte

```diff
@@ -117,7 +118,7 @@
 			<InfiniteScroll on:nnsIntersect={onIntersect} disabled={disableInfiniteScroll}>
 				{#each $icTransactions as transaction, index (`${transaction.data.id}-${index}`)}
 					<li in:slide={{ duration: transaction.data.status === 'pending' ? 250 : 0 }}>
-						<IcTransaction transaction={transaction.data} />
+						<IcTransaction transaction={transaction.data} token={$token ?? ICP_TOKEN} />
```

Comment:
ckErc20 is 6 or 18 decimals 🫠

Reply:
ah! argh! mmm thinking.... not really sure what to choose... render only when `token` is defined? either way, before it was Ethereum for any ICRC eheh!

What do you suggest?

Reply:
Let's keep ICP in this PR and think about it. Rendering when token is defined does not solve it neither when I think about it too long because then, it's also what we discussed earlier, the decimals of the token might not match the amount.